### DISCUSSION TITLE
Data base

### DISCUSSION THREAD

**Discussion by gilberto85** [bcorfman/raven-checkers#25] [General]
does this uses any kind of database like checkerboard ?

**Comment by bcorfman** [comment]
Raven doesn't have a database yet. I was considering that as my next task after finishing up the PDN load/save branch (that should be done this week). If you're interested and want to contribute to hooking up an endgame database, it would be welcome. One huge benefit for it is being able to definitively determine whether the game has ended or not.

**Comment by gilberto85** [comment]
unfortunately i am not a developer but in fiertz web page im sure you can get the DB and End games

**Comment by gilberto85** [comment]
btw is a really good project thank you for your effort !

**Reply by bcorfman** [reply]
Thanks, I'm glad you like it. I didn't come back to the code for several years, but I'm engaged with it again. At some point (soonish) I will make it into a mobile app. Getting enough spare time for development is always the issue.

**Reply by gilberto85** [reply]
thats a good idea man , i am working on my own checkers website ( actually paying ) but this project its really interesting and it can help on my project !!
